setup.cfg

```diff
@@ -1,6 +1,3 @@
-[bdist_wheel]
```

Comment:
According to the [wheel documentation](https://wheel.readthedocs.io/en/stable/user_guide.html), a universal wheel is defined as:

> If your project contains no C extensions and is expected to work on both Python 2 and 3, you will want to tell wheel to produce universal wheels by adding this to your setup.cfg file:

With this change, Twine no longer works on **both** Python 2 and 3, so I believe it is incorrect to declare the wheel as universal. Just being pure Python is not enough to be declared universal.

For an example from another project, Django dropped universal when dropping Python 2. See commit:

https://github.com/django/django/commit/1c466994d9729a8902d34e277a3a685bef2a08e1

Reply:
One idea would be to add a check:

```py
if sys.version_info < (3,4):
    raies ...
```
 This would prevent someone from creating a Python2 wheel. WDYT?

Reply:
I made a suggestion with the setuptools project that this could be handled more centrally by `python_requires`: https://github.com/pypa/setuptools/issues/1633

Reply:
I’d try to avoid imperative code in setup.py. I do agree that if setuptools could derive this information from other signals that would be better. Happy to keep this change as-is.